go.mod

```
@@ -194,6 +194,7 @@ require (
 	go.uber.org/goleak v1.3.0 // @grafana/grafana-search-and-storage
 	go.uber.org/zap v1.27.0 // @grafana/identity-access-team
 	gocloud.dev v0.42.0 // @grafana/grafana-app-platform-squad
+	gocloud.dev/secrets/hashivault v0.42.0 // @grafana/grafana-app-platform-squad
```

Because this is a different package I'd argue it should be own by us
```suggestion
	gocloud.dev/secrets/hashivault v0.42.0 // @grafana/grafana-operator-experience-squad
```